src/SignalR/clients/ts/FunctionalTests/tsconfig.jest.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`ts-jest` changed something with how they compiled files which made the types now actually needed, so had to add a new config file to have the correct `typeRoots` and use that for FunctionalTests

Reply:
👍